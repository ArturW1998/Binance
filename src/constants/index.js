export const ACTION_TYPES = {
  UPDATE_MARKET_PAIRS: 'UPDATE_MARKET_PAIRS',
  SET_ACTIVE_MARKET: 'SET_ACTIVE_MARKET',
  TOGGLE_SOCKET_STREAMS: 'TOGGLE_SOCKET_STREAMS',
};

export const PATHS = {
  BASE_PATH: 'wss://stream.binance.com',
  STREAM_PATH: '/stream',
  STREAM_PARAM: 'streams=',
};

export const LISTS = {
  PAIRS_LIST: ['BNB', 'BTC', 'ETH', 'USDT'],
  DOTS_LIST: [{ id: 1 }, { id: 2 }, { id: 3 }, { id: 4 }],
  TABLE_DATA_LIST: [
    { title: 'Pair' },
    { title: 'Latest Price' },
    { title: 'Open' },
    { title: 'High' },
    { title: 'Low' },
    { title: 'Volume' },
  ],
};

export const CONFIG = {
  BASE_NAME: '/Binance/',
  LOCALSTORAGE_KEY: 'markets',
};
