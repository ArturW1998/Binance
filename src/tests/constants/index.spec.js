import { ACTION_TYPES } from '../../constants';

describe('constants', () => {
  it('ACTION_TYPES', () => {
    expect(ACTION_TYPES).toMatchSnapshot();
  });
});
