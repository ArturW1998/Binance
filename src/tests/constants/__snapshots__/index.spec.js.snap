// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants ACTION_TYPES 1`] = `
Object {
  "SET_ACTIVE_MARKET": "SET_ACTIVE_MARKET",
  "TOGGLE_SOCKET_STREAMS": "TOGGLE_SOCKET_STREAMS",
  "UPDATE_MARKET_PAIRS": "UPDATE_MARKET_PAIRS",
}
`;
